tests/tasks/test_llm_integration.py

```
@@ -0,0 +1,512 @@
+        mock_llm = Mock()
+        mock_chunk = Mock()
+        mock_chunk.content = "Hello"
+        mock_llm.stream.return_value = [mock_chunk]
+        
+        self.mock_task.get_llm.return_value = mock_llm
+        
+        messages = [{"role": "user", "content": "Hello"}]
+        metadata = {"watch_mode": True, "speaker": "Assistant"}
+        
+        with patch('tasks.llm.track_streaming_chunks'):
+            result = stream_chat_response(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)